content/docs/csidriver/installation/operator/_index.md

```diff
@@ -263,6 +296,7 @@ The CSI Drivers installed by the Dell CSI Operator can be updated like any Kuber
     ```
     and modify the installation
 * Modify the API object in-place via `kubectl patch`
+* Modify the API object via `kubectl apply -f <file>` . Refer [here](https://github.com/dell/dell-csi-operator/tree/master/samples) to create a CR (Custom Resource) using the sample files provided.
 
```

Comment:
@randeepsharma : Modify the API object via `kubectl apply -f <file>` . **There is a space after the backtick, Please remove the same**

Reply:
The above line is removed based on Rensy's comment.